packages/i18n/src/locales/en/translations.json

```diff
@@ -287,7 +287,7 @@
   "Cancel": "Cancel",
   "edit": "Edit",
   "archive": "Archive",
-  "restor": "Restore",
```

Comment:
Can you also check for this typo in all other translation files?

Reply:
sure

Reply:
You were right @prateekshourya29  we had this typo in other files as well I fixed it.

Reply:
Thanks!